pinot-common/src/main/java/org/apache/pinot/pql/parsers/PinotQuery2BrokerRequestConverter.java

```diff
@@ -272,7 +271,11 @@ private FilterQuery traverseFilterExpression(Expression filterExpression, Filter
             filterQuery.setColumn(ParserUtils.standardizeExpression(operands.get(0), false));
             filterQuery.setValue(ParserUtils.getFilterValues(filterKind, operands));
             break;
-
+          case IS_NULL:
+          case IS_NOT_NULL:
+            //first operand is the always the column
```

Comment:
(nit) Change line 270 as well
```suggestion
            // First operand is the left-hand side column/expression of the predicate
```